azure-storage.js.html

```diff
@@ -493,7 +493,7 @@ <h4 class="modal-title">Search results</h4>
 <span class="jsdoc-message">
 	Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc 3.4.3</a>
 	
-		on Mon Sep 18th 2017
+		on Tue Dec 5th 2017
```

Comment:
We would better consider to update the jsdoc template, according to internal team's suggestion.

Reply:
Yeah. Some contents are covered by our templates 'index'. It's a bug of our template, I opened an issue for them. At the same time, let's try to find out some candidate templates which we can consider to update.

Reply:
https://github.com/docstrap/docstrap/issues/319